ui/src/authorizations/utils/permissions.ts

```diff
@@ -59,6 +58,13 @@ const ensureT = (orgID: string) => (t: PermissionTypes) => {
           resource: {type: t, orgID},
         },
       ]
+    case 'orgs':
```

Comment:
Can you add a  comment here noting specifically that we expect the org permission to be at `id` and not to have `orgID`?